### PR TITLE
init project

### PR DESCRIPTION
**Pull request by suuhv**
[x] eslint: failed
[o] rails_best_practices: success
[o] rspec: success
[o] scss-lint: success
[o] brakeman: success
[o] reek: success
[o] bundle-audit: success
[o] rubocop: success

### CODE REVIEW

**Comment by suuhv** [comment]
ready

File: .framgia-ci-result.temp.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+brakeman:
```
Comment (dieunb) [inline .framgia-ci-result.temp.yml]: ignore file này

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,77 @@
+source 'https://rubygems.org'
```
Comment (dieunb) [inline Gemfile]: nháy kép

**Comment by dieunb** [comment]
commented-vô review lại code đi em

**Comment by suuhv** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Project2Suu</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: user double quote

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: user double quote

File: app/views/layouts/mailer.html.erb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+      /* Email styles need to be inline */
```
Comment (thangtx88) [inline app/views/layouts/mailer.html.erb]: bỏ comment đi

**Comment by thangtx88** [comment]
commented

**Comment by suuhv** [comment]
ready

**Comment by huongnguyenmta** [comment]
commented-replace all single quote => double quote-no

**Comment by suuhv** [comment]
ready

**Pull request merged**